src/runtime/virtcontainers/pkg/annotations/annotations.go

```diff
@@ -249,6 +249,9 @@ const (
 	// GuestSeLinuxLabel is a SELinux security policy that is applied to a container process inside guest.
 	GuestSeLinuxLabel = kataAnnotRuntimePrefix + "guest_selinux_label"
 
+	// GuestAppArmorProfile is an AppArmor profile that is applied to a container process inside guest.
+	GuestAppArmorProfile = kataAnnotRuntimePrefix + "guest_apparmor_profile"
```

Comment:
Better to add this to `docs/how-to/how-to-set-sandbox-config-kata.md` if it can be configured by an annotation.

Reply:
Thanks for the information. I added the description and modified this feature for users to be able to enable it using the pod-level annotation.